flow/record/adapter/sqlite.py

```diff
@@ -195,13 +196,14 @@ def read_table(self, table_name: str) -> Iterator[Record]:
 
     def __iter__(self) -> Iterator[Record]:
         """Iterate over all tables in the database and yield records."""
+        ctx = get_app_context()
+        selector = self.selector
         for table_name in self.table_names():
             self.logger.debug("Reading table: %s", table_name)
             for record in self.read_table(table_name):
-                if not self.selector or self.selector.match(record):
+                if match_record_with_context(record, selector, ctx):
                     yield record
 
-
```

Comment:
Won't this fail linting? Thought you needed two empty whitespaces between classes.

Reply:
There's a `ruff format --check flow tests` missing in the `tox.ini` config for linting.